modules/alb/variables.tf

```diff
@@ -222,6 +222,12 @@ variable "dns_delegated_component_name" {
   description = "Atmos `dns-delegated` component name"
 }
 
+variable "dns_delegated_environment_name" {
+  type        = string
+  default     = "gbl"
```

Comment:
I would like this to default to `null`, and when null, use `module.iam_roles.global_environment_name`.